### PR TITLE
Add basic helm chart for celery

### PR DESCRIPTION
**Pull request by necromancerthedark**
## Description

This pull request attempts to create a helm chart which can be used for deploying celery with ease on kubernetes, it is highly customizable and can be changed in own ways, will be continuously improving it in upcoming time.

### CODE REVIEW

**Comment by thedrow** [comment]
The linter yaml in precommit is failing. Please check the output and fix the issues it points out to.

File: helm-chart/templates/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: {{ .Values.secrets.name }}
+  labels:
+    app: {{ include "..fullname" . }}
+type: Opaque
+data:
+  {{- range $key, $value := .Values.secrets.data }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+{{- end }}
```
Comment (auvipy) [inline helm-chart/templates/secret.yaml:13]:
```suggestion
{{- end }}

```

File: helm-chart/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
```
Comment (auvipy) [inline helm-chart/templates/deployment.yaml:69]:
```suggestion
      {{- end }}
      
```

File: helm-chart/templates/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.configmap.name }}
+  labels:
+    app: {{ include "..fullname" . }}
+data:
+{{- .Values.configmap.data | toYaml | nindent 2 }}
```
Comment (auvipy) [inline helm-chart/templates/configmap.yaml:8]:
```suggestion
{{- .Values.configmap.data | toYaml | nindent 2 }}

```

**Review by auvipy — CHANGES_REQUESTED** [review]

**Comment by necromancerthedark** [comment]
@thedrow added exclusion for helm chart in check-yaml, 
took reference from this pre-commit issue: https://github.com/pre-commit/pre-commit-hooks/issues/420

**Review by auvipy — CHANGES_REQUESTED** [review]
can we add basic introductory documentation for how to use this chart for both local / deployment? including appropriate links for better understanding for people willing to try it?

**Comment by necromancerthedark** [comment]
sure,
one question: should Readme for this go inside helm chart repo or somewhere outside.
My opinion would be inside the helm-chart folder only

**Comment by auvipy** [comment]
lets start with readme inside this repo first.... in future we will determine should this be recommended in deployment docs in celery

**Pull request merged**